common/hook.h

```diff
@@ -21,6 +21,8 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #include "keyboard.h"
 #include "led.h"
 
+typedef struct _host_driver_t host_driver_t;
+
```

Comment:
Why do you do this instead of just including host_driver.h?
Is there programming theory or something?